lib/capybara/selenium/driver.rb

```diff
@@ -455,6 +459,10 @@ def find_context
     browser
   end
 
+  def active_element
+    build_node evaluate_script("document.activeElement")
```

Comment:
Pretty sure evaluate_script already calls `build_node` when returning  elements

Reply:
You're right: https://github.com/teamcapybara/capybara/blob/22252382421f3126d6e10fa90a63237ff817436a/lib/capybara/selenium/driver.rb#L441-L452